bf2fc6cc711aee1a0c2a/kas-fleet-manager
# feat: update observability operator to v3.0.15

## Description

Update Observability Operator to v3.0.15 to support GCP and allow the operand pods to be moved to other nodes to improve dynamic scaling.

## Verification Steps

1. Run KFM and terraform an OSD cluster
2. Make sure the observability namespace is populated
3. Grab the cluster_id from the Observability resource status and query stage Observatorium for it
4. Make sure metrics are received from the cluster
